docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -855,6 +895,24 @@ IMPORTANT: All plugins have a runtime dependency on the `logstash` core gem, and
 a development dependency on `logstash-devutils`.
 
 [float]
+==== Jar dependencies
+In some cases, such as the
+https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/logstash-output-elasticsearch.gemspec#L22-L23[Elasticsearch output plugin],
+your code may depend on a jar file.  In cases such as this, the dependency is
+added in the gemspec file in this manner:
+
+[source,ruby]
+[subs="attributes"]
+----------------------------------
+  # Jar dependencies
+  s.requirements << "jar 'org.elasticsearch:elasticsearch', '1.4.0'"
+  s.add_runtime_dependency 'jar-dependencies'
```

Comment:
Is this runtime dep required? JRuby ships with jar-dependencies and logstash-devtools also requires it, so plugins probably don't' need it? Right?